client/packages/common/src/intl/locales/en/common.json

```diff
@@ -411,14 +414,14 @@
   "link.copy-to-clipboard": "Copy to Clipboard",
   "log.changed-from": "From",
   "log.changed-to": "to",
-  "log.invoice-created": "Shipment created",
-  "log.invoice-deleted": "Shipment deleted",
-  "log.invoice-number-allocated": "Shipment number allocated",
-  "log.invoice-status-allocated": "Shipment allocated",
-  "log.invoice-status-delivered": "Shipment delivered",
-  "log.invoice-status-picked": "Shipment picked",
-  "log.invoice-status-shipped": "Shipment shipped",
-  "log.invoice-status-verified": "Shipment verified",
+  "log.invoice-created": "Created",
+  "log.invoice-deleted": "Deleted",
+  "log.invoice-number-allocated": "Invoice number allocated",
```

Comment:
Are we showing the word `invoice` in UI? 🤔

Reply:
Felt like it made the most sense in this one place... could just be `Number allocated` - but as mentioned below that felt super vague??

Reply:
No, we aren't using `Invoice` and I'm reluctant to introduce it now. The list has this:

<img width="192" alt="image" src="https://github.com/msupply-foundation/open-msupply/assets/9192912/7b07b24b-3e72-4088-92a6-9890c727a39b">

with the label as `number` or `shipment number` - given that the number isn't shown on the Return detail view I don't see why the Inbound & outbound shipments should suffer b/c Returns are reusing status enums

I'm fine with returns saying shipment in this case, though `Number allocated` would also be ok.
The least preferred option is `Invoice number allocated` 😜

Reply:
![Screenshot 2024-05-20 at 4 47 07 PM](https://github.com/msupply-foundation/open-msupply/assets/55115239/b843ec43-ddf5-4467-89b7-069890c791c2)

Oh returns list views actually do have that...